['transform_resize(600)', 'transform_pad(600)']
none
none
Transforms:
['transform_resize(600)', 'transform_pad(600)'] none none
Compose(
    <transforms.resize.TransformResize object at 0x16a238a10>
    <transforms.pad.TransformPad object at 0x16a243b90>
) None None
Initialization took 3.6880977153778076 seconds
Traceback (most recent call last):
  File "/Users/volodimir/Downloads/Bachelor/CrossValidation/training.py", line 95, in <module>
    train()
  File "/Users/volodimir/Downloads/Bachelor/CrossValidation/training.py", line 74, in train
    trainer.train_epoch()
  File "/Users/volodimir/Downloads/Bachelor/CrossValidation/utils/trainer.py", line 45, in train_epoch
    for images, labels in self.train_loader:
  File "/Users/volodimir/anaconda3/envs/Global/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
           ^^^^^^^^^^^^^^^^^
  File "/Users/volodimir/anaconda3/envs/Global/lib/python3.11/site-packages/torch/utils/data/dataloader.py", line 675, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/volodimir/anaconda3/envs/Global/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/volodimir/anaconda3/envs/Global/lib/python3.11/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
            ~~~~~~~~~~~~^^^^^
  File "/Users/volodimir/Downloads/Bachelor/CrossValidation/data/dataset.py", line 75, in __getitem__
    image = read_image(frame_path)
            ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/volodimir/anaconda3/envs/Global/lib/python3.11/site-packages/torchvision/io/image.py", line 258, in read_image
    data = read_file(path)
           ^^^^^^^^^^^^^^^
  File "/Users/volodimir/anaconda3/envs/Global/lib/python3.11/site-packages/torchvision/io/image.py", line 52, in read_file
    data = torch.ops.image.read_file(path)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/volodimir/anaconda3/envs/Global/lib/python3.11/site-packages/torch/_ops.py", line 755, in __call__
    return self._op(*args, **(kwargs or {}))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: [Errno 2] No such file or directory: '/Volumes/LaCie/Macbook/Bachelor/CrossValidation/cropped/video_frames/010_image_64754365864107.mp4/frame_50.png'